langlib/langlib-test/src/test/resources/test-src/langlib_test_negative.bal

```diff
@@ -38,6 +38,24 @@ function testEnsureTypeWithUnion1() returns error? {
     int|string|float[] name = check j.name;
 }
 
+function testToStringNegative() {
+    json j = {
+        name : "Name",
+        address : {
+            country : "Country",
+            city : "City"
+        }
+    };
+
+    var x1 = j.age.toString();
+    var x2 = j.address.town.toString();
+    var x3 = foo().toString();
```

Comment:
Do we have negative tests for `toBalString`?

Reply:
Added